index.html

```diff
@@ -1,28 +1,44 @@
-<!DOCTYPE HTML >
-<html>
-<head>
-	<title>The network</title>
-	<link rel="stylesheet" type="text/css" href="normalize.css" />
-	<link rel="stylesheet" type="text/css" href="styles.css" />
-</head>
-<body>
-	
-	<div id="header">
-		<div id="logo">
-			<img src="images/logo.png" alt="the network"/>
-		</div>	
-		<div id="search">
-			<form method="get">
-				<input type="text" name="post" placeholder="thoughts?"/>
-				<input type="submit" value="Post">
-			</form>
-		</div>
-		<br>
-	</div>
-	
-	<div id="page">
-
-	</div>
-
-</body>
-</html>
+<!DOCTYPE HTML >
+<html>
+<head>
+	<title>The network</title>
+	<link rel="stylesheet" type="text/css" href="normalize.css" />
+	<link rel="stylesheet" type="text/css" href="styles.css" />
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+	<script>
+		$(document).ready(function(){
+		$('input[value=Post]').on('click',function(event){ /* Check if form was submitted */
+			var $thoughts = $("#search").find('input').val(); /* Find input field data */
+			var dNow = new Date(); /* Set Date Time */
+			var utc = new Date(dNow.getTime() + dNow.getTimezoneOffset() * 60000)
+			var utcdate= (utc.getMonth()+1) + '/' + utc.getDate() + '/' + utc.getFullYear() + ' ' + utc.getHours() + ':' + utc.getMinutes() + ':' + utc.getSeconds();
+			var $day = 	utc.getSeconds();  /* Find seconds */

```

Comment:
I think a slightly less complicated way of getting the current time's seconds would be
let seconds= new Date().getSeconds();
or
let seconds = dNow.getSeconds();

Reply:
Your variable name is a little confusing as well. Typically prepending a variable with $ indicates its an jQuery instance. Also it seems like you are assigning it the number of seconds but the variable name is $day.

Reply:
Much less complicated

Reply:
Thanks for the review Lorenzo. I appreciate it. Any other critiques?